packages/generate-scaffold/template/utils/dev-config.js

```diff
@@ -0,0 +1,16 @@
+const createCommonsChunkPlugin = require("@webpack-cli/webpack-scaffold").commonChunksPluginCreate;
```

Comment:
We do not support CommonsChunk

Reply:
@evenstensberg  should I use the default `SplitChunksPlugin`?

Reply:
I added a splitchunk in optimization 👍

Reply:
done 👍